docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -1252,12 +1252,12 @@ or being killed.
 
 Query Parameters:
 
--   **dockerfile** - path within the build context to the Dockerfile. This is 
+-   **dockerfile** - path within the build context to the Dockerfile. This is
```

Comment:
I _think_ the whitespaces in this file are intentional as it signifies a new line for the markdown renderer.